react/Avatar/Readme.md

```diff
@@ -22,7 +22,7 @@ import palette from 'cozy-ui/transpiled/react/palette';
 ### With disabled color
 
 ```jsx
-import cozyLogo from '../../docs/cozy-logo_white_128.png'
+import cozyLogo from 'cozy-ui/transpiled/react/../docs/cozy-logo_white_128.png'
```

Comment:
Same here, everything inside docs folder is not supposed to be in the public API, so original import was better for me

Reply:
<img width="921" alt="Capture d’écran 2021-12-24 à 14 23 35" src="https://user-images.githubusercontent.com/8363334/147355796-99af4843-56e8-4683-9eb5-5b243d3a82ef.png">